Gemfile.lock

```diff
@@ -121,13 +121,9 @@ GEM
       sentry-raven (>= 2.7.1, < 2.12.0)
       statsd-ruby (~> 1.4.0)
       unicorn (>= 5.4, < 5.6)
-    govuk_frontend_toolkit (8.2.0)
```

Comment:
Just curious - looks like toolkit is being removed from here, but there's no change in the Gemfile and no mention of removing toolkit in the PR. Is this deliberate?

Reply:
The quotes changed under my nose between me producing this branch and trying to check it in (re-implemented linting rules) and my rebase has clearly gone wrong somewhere.

Reply:
I'm assuming that toolkit was a dependency brought in by the publishing components, which has gone away with the updated version. The branch has been deployed to integration again and all looks fine.